joomla/joomla-cms
# [4.0] ComponentAdapter prepared statements

### Summary of Changes

Adds some missing prepared statements, cleans up queries and fixes removing categories when uninstalling components.

### Testing Instructions

Try installing and uninstalling a component (e.g. `com_weblinks`).
Check that related entries are removed from `#__categories` tables.

### Expected result

Entries removed.

### Actual result

Entries not removed.

### Documentation Changes Required

No.
